custom_components/powercalc/data/signify/LTP008/model.json

```diff
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Signify, LTP008",
```

Comment:
Can you review this measure device? Doesn't seem correct. Supposed to be Shelly Plug S?